changelog.d/+progress_listeners.fixed.md

```diff
@@ -0,0 +1 @@
+Fix closing of passed progress listeners in `Bucket.upload` and `Bucket.copy`
```

Comment:
you have "fixed" download as well right?

Reply:
I "don't" think I "have" "fixed" any public "methods" in that regard

Reply:
you changed have Emerger, so download is also affected, right?

Reply:
ah it does download sometimes, but only if the ultimate goal is to download and then upload

Reply:
but I don't think I touched that part